pkg/networking/cilium/upgrade_plan.go

```diff
@@ -46,6 +61,8 @@ type ComponentUpgradePlan struct {
 	UpgradeReason string
 	OldImage      string
 	NewImage      string
+	OldEnforcementPolicy string
+	NewEnforcementPolicy string
```

Comment:
these fields might not make a lot of sense anymore since the config doesn't deal with versions and CM and deployments don't deal with policy enforcement. The abstraction has broken a bit.

Reply:
Yeah agreed. I just went with the simplest solution first. We still do need some notion of "this is what it is now" and "this is what we want it to be" though.